TestBenches/FileReadUtility.h

```diff
@@ -256,9 +256,9 @@ unsigned int compareMemWithMemPage(const MemType& memory_ref, const MemType& mem
   constexpr int lsb = (LSB >= 0 && MSB >= LSB) ? LSB : 0;
   constexpr int msb = (LSB >= 0 && MSB >= LSB) ? MSB : MemType::getWidth() - 1;
 
-  for (unsigned int ipage = 0; ipage < 4; ++ipage) {
+  for (unsigned int ipage = 0; ipage < memory_ref.getNPage(); ++ipage) {
     //FIXME
-    for (unsigned int i = 0; i < memory_ref.getDepth()/4; ++i) {
+    for (unsigned int i = 0; i < memory_ref.getDepth(); ++i) {
```

Comment:
@tomalin This works for the SW, where the number of pages is one, but generally, shouldn't the upper bound of the loop still be divided by the number of pages?

```suggestion
    for (unsigned int i = 0; i < memory_ref.getDepth()/memory_ref.getNPage(); ++i) {
```

Reply:
That makes sense. Thanks for clarifying.